examples/mediatek/README.md

```diff
@@ -0,0 +1,29 @@
+# ExecuTorch Neuron Backend examples
+
+## Directory structure
```

Comment:
Can you add pre-requisite here as to which phones support MTK chipset? ALso do add "validated on" to indicate which phone or chipset this is validated on

Reply:
Thanks for your suggestion! We have added the list of supported chips that the examples are validated on. Also, we have refactor this README for better readability.